### PR TITLE
Unifies release variable type across the project

### PR DESCRIPTION
**Pull request by agileshaw**
Change the return type of `OpenStackRelease.next_release` and `OpenStackRelease.previous_release` from `str` to an `OpenStackRelease` object. This unifies the target release variable type across the project.

Supersedes: #120

### CODE REVIEW

File: cou/utils/openstack.py [hunk 1]
```diff
@@ -149,6 +150,8 @@
     "lunar": "antelope",
 }
 
+SOME_TABLE = {"focal-yoga": True}
```
Comment (gabrielcocenza) [inline cou/utils/openstack.py:153]: Did you miss to remove this line?

**Review by gabrielcocenza — COMMENTED** [review]
Very small detail, but LGTM

File: cou/utils/openstack.py (hunk 1 above)
Comment (agileshaw) [reply cou/utils/openstack.py]: Indeed :laughing: Thanks for catching it.

**Review by gabrielcocenza — APPROVED** [review]

**Review by rgildein — APPROVED** [review]
LGTM

**Review by gabrielcocenza — APPROVED** [review]

**Review by rgildein — APPROVED** [review]
LGTM

**Pull request merged**